app/tests/evaluation_tests/test_forms.py

```diff
@@ -127,7 +125,7 @@ def test_algorithm_queryset(self):
 
         assert alg1 in form.fields["algorithm"].queryset
         assert alg2 not in form.fields["algorithm"].queryset
-        assert alg2 not in form.fields["algorithm"].queryset
+        assert alg3 not in form.fields["algorithm"].queryset
```

Comment:
Not relevant for this PR, but I noticed this typo in this test.

Reply:
Thanks for highlighting it!